vignettes/adrs.Rmd

```diff
@@ -599,21 +655,24 @@ adrs <- adrs %>%
     )
   )
 
-confirmed_resp <- date_source(
-  dataset_name = "adrs",
-  date = ADT,
-  filter = PARAMCD == "CRSP" & AVALC == "Y"
-)
-
 adrs <- adrs %>%
-  derive_param_clinbenefit(
-    dataset_adsl = adsl,
-    filter_source = PARAMCD == "OVR" & ANL01FL == "Y",
-    source_resp = confirmed_resp,
-    source_pd = pd,
-    source_datasets = list(adrs = adrs),
-    reference_date = RANDDT,
-    ref_start_window = 42,
+  derive_extreme_event(
+    by_vars = exprs(STUDYID, USUBJID),
+    order = exprs(desc(AVALC), ADT, PARAMCD),
+    mode = "first",
+    events = list(
+      event(
```

Comment:
same comment as above - why not create a `crsp_y` for this?

Reply:
See my comment above.